sites/website/src/docs/components/getting-started.md

```diff
@@ -95,7 +95,7 @@ For simplicity, examples throughout the documentation will assume the library ha
 
 ## Add components
 
-With the components registered, add any component to the HTML. That's it! For a comprehensive list of all elements, see the [Components](/docs/components) section.
+With the components registered, add any component to the HTML. That's it! For a comprehensive list of all elements, see the [Components](/docs/components/accordion) section.
```

Comment:
This is an interesting problem. Does docusaurus allow for a containing menu item to have it's own page?

Reply:
I don't know actually. I realized the current link was broken so made this quick fix. I'll add a follow-up to see if we can configure a landing page for the group itself in a future update.